Week07/src/Main.java

```diff
@@ -0,0 +1,79 @@
+import java.util.*;
+
+public class Main {
+
+    private static Object Hobby;
+
+    public static void main(String[] args) {
+
+        TreeSet<Persoana> Personaje = new TreeSet<>(Comparator.comparing(Persoana::getName).thenComparing(Persoana::getAge));
```

Comment:
It would be better to program against interfaces instead of implementations.
In this case, you should use `Set` as type for the variable.
Same goes to the following variables too.

Also, please use naming conventions for variables. Their names should start with lower case letters etc.